src/renderer/src/views/Migration/index.tsx

```diff
@@ -138,15 +147,41 @@ const messages = defineMessages({
   migratePreviousInstruction: {
     id: 'views.migration.migratePreviousInstruction',
     defaultMessage:
-      'We have found a previous Mapeo project. Follow the steps below to begin migrating to the new Mapeo:',
+      'We have found a previous Mapeo project. To begin using New Mapeo with the data you have on old Mapeo, data migration is required. Once you are ready to migrate your data to new Mapeo, follow the steps below.',
   },
   migrateInstruction1: {
     id: 'views.migration.migrateInstruction1',
-    defaultMessage: 'Migrating from [x] to [y]',
+    defaultMessage:
+      'Migrating from Mapeo5 to MapeoSuperS will add additional security features including encryption',
   },
   migrateInstruction2: {
     id: 'views.migration.migrateInstruction2',
-    defaultMessage: 'You will have to coordinated with your team to upgrade each mobile device to {appTitle}',
+    defaultMessage:
+      'If you already have peer you sycronize Mapeo with, it is important that you are all ready to update together. You will not be able to syncronize your new mapeo data with a devise thet has old Mapeo',
```

Comment:
Not grammatical so feel free to ignore:

```suggestion
      'If you already have peer you sycronize Mapeo with, it is important that you are all ready to update together. You will not be able to syncronize your MapeoSuperS data with a devise thet has Mapeo 5',
```